setup.cfg

```diff
@@ -19,27 +19,27 @@ include_package_data = True
 package_dir =
     = src
 packages = find_namespace:
-python_requires = >=3.8,!=3.9.7,<3.11
+python_requires = >=3.9,!=3.9.7,<3.11
 
 install_requires =
-    orquestra-quantum==0.9.0
-    orquestra-qiskit==0.9.0
-    orquestra-cirq==0.8.0
+    orquestra-quantum~=0.9.0
+    orquestra-qiskit~=0.9.0
+    # temporary hack until newest version of orquestra-cirq is released
+    orquestra-cirq @ git+https://github.com/zapatacomputing/orquestra-cirq.git
```

Comment:
Please pin to a specific commit, this might change whenever someone pushes to `main`.

Reply:
I think I'm just going to wait until we publish the latest version of `orquestra-cirq` to finish this PR.